runtime/src/main/java/org/corfudb/runtime/collections/StreamListener.java

```diff
@@ -0,0 +1,42 @@
+package org.corfudb.runtime.collections;
+
+/**
+ * This is the callback interface that any client subscribe()ing to CorfuStore updates must implement.
+ *
+ * Created by hisundar on 2019-10-18
+ */
+public interface StreamListener {
+    /**
+     * A corfu update may have multiple updates belonging to different streams.
```

Comment:
A corfu update can/may have multiple entries belonging to different streams.

Reply:
done.